DAQ/src/SConscript

```diff
@@ -18,14 +18,13 @@ rootlibs = env['ROOTLIBS']
 
 mainlib = helper.make_mainlib ( [ 'mu2e_ConditionsService',
                                   'mu2e_GeometryService',
-                                  'mu2e_MCDataProducts',
+                                  # 'mu2e_MCDataProducts',
```

Comment:
It's great that we no longer need MCDataProducts - thanks for looking after that. Please just remove the line instead of comment it out.  Same comment for any other place this shows up.